frontend/public/locales/en/terms-and-conditions.json

```diff
@@ -0,0 +1,63 @@
+      "terms-and-conditions-of-use-disclaimers-bullet-one": "any matters or factors outside of its control, including the availability or unavailability of the internet, or telecommunications or other infrastructure systems; or",
+      "terms-and-conditions-of-use-disclaimers-bullet-two": "the non-acceptance of information provided through the Online Application; or",
+      "terms-and-conditions-of-use-disclaimers-bullet-three": "non-compliance with any of these Terms.",
+      "terms-and-conditions-of-use-disclaimers-text-continued": "I hereby agree to indentify and hold ESDC harmless from any such liability.",
+      "terms-and-conditions-of-use-changes-to-these-terms-of-use-title": "Changes to these Terms of Use",
+      "terms-and-conditions-of-use-changes-to-these-terms-of-use-text": "ESDC may amend these Terms of Use at any time and without notice. Such changes shall be effective immediately on the date they are published. By using the Online Application, you agree to abide by the amended Terms. You are responsible to ensure you understand and retain the amended Terms.",
+      "privacy-notice-statement-title": "Privacy Notice Statement",
+      "privacy-notice-statement-personal-information-title": "Personal Information",
+      "privacy-notice-statement-personal-information-text": "Service Canada, which is part of Employment and Social Development Canada (ESDC),administers applications for the Canada Dental Care Plan (CDCP) on behalf of Health Canada (HC). Service Canada will collect the personal information you include in your CDCP application to validate eligibility against the CDCP criteria.",
+      "privacy-notice-statement-how-we-protect-your-privacy-title": "How we protect your privacy",
+      "privacy-notice-statement-how-we-protect-your-privacy-text-part-one": "The collection and use of your personal information is authorized under the Privacy Act, the Department of Health Act, the Dental Care Measures Act, the Department of Employment and Social Development Act, and other applicable laws.",
+      "privacy-notice-statement-how-we-protect-your-privacy-text-part-two": "You have the right to the protection of, access to, and where applicable, the correction of your personal information. It is kept in the following personal information banks:",
+      "privacy-notice-statement-how-we-protect-your-privacy-bullet-one": "[LIST PIBS WITH THEIR NUMBERS]",
+      "privacy-notice-statement-how-we-protect-your-privacy-text-part-three-part-1": "You can learn how to access this information in the government publication called ",
+      "privacy-notice-statement-how-we-protect-your-privacy-text-part-three-part-1-link-text" : "Info Source. ", 
+      "privacy-notice-statement-how-we-protect-your-privacy-text-part-three-part-2": " You can also access Info Source online at any Service Canada Centre.",
```

Comment:
Consider using the the `<Trans>` component instead of breaking the i18n into multiple parts.

See https://react.i18next.com/latest/trans-component
(There are also a few occurrences of where it is used currently in CDCP)

Reply:
Fixed.